packages/data-store/src/entities/claim.ts

```diff
@@ -22,19 +22,19 @@ export class Claim extends BaseEntity {
     onDelete: 'CASCADE',
   })
     //@ts-ignore
-  issuer: Identifier
+  issuer: Relation<Identifier>
```

Comment:
@mirceanis wanted to specifically call out this change. We might want to manually test a migration to this version to make sure there are no problems?